Changelog.md

```diff
@@ -5,6 +5,9 @@ Changelog
 
 * Use ceil in `ResettingController` for a better token lifetime approximation
 * Removed unused translation keys
+* Removed form deprecations
+* Added `symfony/templating` as a dependency
```

Comment:
Why drop the requirement in FOSUserBundle? If it's needed, I don't see why one would drop support for it 😕 
I think I don't get it 😕